documentation/tutorials/sensitivity/PEcAn_sensitivity_tutorial_v1.0.Rmd

```diff
@@ -25,7 +25,13 @@ RunDir <- "PEcAn_99000000002"
 
 Within that folder, you will find the *run/* folder that has a folder for every SIPNET run and the *out/* folder, that contains the output from each of these runs. The nice thing is that PEcAn has also produced “.RData” files of these model outputs that can be easily read into R by just clicking on them!
 
-So let’s try to compare Ameriflux NEE to SIPNET NEE across all these runs to make a plot of parameter vs. goodness-of-fit. We’ll start with root mean square error (**RMSE**), but then discuss some other tests, too.
+To do some automatic querying of data, let's open up a connection to the Bety database. 
+```{r, eval = FALSE}
+bety<-list(host = "psql-pecan.bu.edu", driver = "PostgreSQL", user = "bety", dbname = "bety", password = "bety")
```

Comment:
1) For the tutorial, they'll be using their localhost not psql-pecan
2) Probably best to get these settings from the settings file, that makes the tutorial more machine independent.
3) As I've told lots of people lots of times, we really need to stop all these redundant calls to src_postgres all over the place. Please implement a wrapper function that just takes bety as an arg and which will ultimately replace the current db_open. Then please grep/replace all these redundant calls.